edgexfoundry/edgex-compose
# feat: Upgrade to eKuiper 1.6

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>


## Testing Instructions
Run non-secure compose
Create a stream
Create a rule
Verify in eKuiper logs that the run created successfully and not errors are occurring as data is flowing.
